Default disguise for each InfantryType (#373)
Phobos-developers/Phobos: docs/New-or-Enhanced-Logics.md

NotHuman.RandomDeathSequence=yes  ; boolean
```

> ### Default disguise for each InfantryType
> - Infantry with `DefaultDisguise` can now has its own default disguise.
>   - This tag's priority is higher than Ares' [Sides]>DefaultDisguise

Reviewer: ```suggestion
### Default disguise for individual InfantryTypes

- Infantry can now have its `DefaultDisguise` overridden per-type.
  - This tag's priority is higher than Ares' per-side `DefaultDisguise`.
```

In `rulesmd.ini`:
```ini

> [SOMEINFANTRY]                    ; InfantryType
> DefaultDisguise=E2                ; InfantryType              

Reviewer: ```suggestion
[SOMEINFANTRY]      ; InfantryType
DefaultDisguise=E2  ; InfantryType              
```
